uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/modules/models.js

```diff
@@ -11,8 +12,9 @@ define([
   return angular.module('leave-absences.models', [
     'common.models',
     'common.services',
+    'angularFileUpload',
```

Comment:
If you have the `FileUpload` service injecting the `common/angular-file-upload` module, why didn't you define the `angularFileUpload` dependency in [`common/modules/services`](https://github.com/civicrm/civihr/blob/la-milestone-3/org.civicrm.reqangular/src/common/modules/services.js) instead of doing it here?